### DISCUSSION TITLE
Welcome to vault-recovery-key Discussions!

### DISCUSSION THREAD

**Discussion by bruj0** [bruj0/vault-recovery-key#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kmahi** [comment]
Hello, does it work wit Transit ?
